docs/modules/ROOT/partials/nav-adrs.adoc

```diff
@@ -35,4 +35,5 @@
 ** xref:adr/0035-ikube-2-0-migration-appcat.adoc[]
 ** xref:adr/0036-revisiting-postgresql-operators.adoc[]
 ** xref:adr/0037-mariadb-bitnami-replacement.adoc[]
-** xref:adr/0038-appcat-redis-alternative.adoc[]
\ No newline at end of file
+** xref:adr/0038-appcat-redis-alternative.adoc[]
+** xref:adr/42-backup-encryption-for-mariadb-galera-and-postgresql.adoc[]
```

Comment:
You will also have to add your document to: docs/modules/ROOT/pages/adr/index.adoc